index.html

```diff
@@ -243,26 +243,43 @@ <h3>Sampling and Reporting Rate</h3>
 
 <section> <h2>Platform primitives</h2>
   <p>
-    The [=platform collector=] refers to a platform interface, with which the [=user agent=] interacts to
-    obtain the telemetry readings required by this specification.
+    A <dfn>pressure source</dfn> is an abstract, [=implementation-defined=]
+    interface to hardware counters or an underlying telemetry framework that
+    provides information about a <dfn>source types</dfn>
+    defined by {{PressureSource}}. A [=pressure source=] can make use of data
+    fusion with data from additional sources if that provides better results.
   </p>
   <p>
-    A [=platform collector=] can be defined by the underlying platform (e.g. in a native telemetry
-    framework) or by the [=user agent=], if it has a direct access to hardware counters.
+    A <dfn>platform collector</dfn> is an abstract interface responsible for
+    obtaining telemetry samples from a [=pressure source=], translating them
+    into [=pressure states=] and providing them to the [=user agent=].
   </p>
   <p>
-    A [=platform collector=] can support telemetry for different <dfn>source types</dfn> of computing
-    devices defined by {{PressureSource}}, or there can be multiple [=platform collectors=].
+    A [=platform collector=] has the following associated data:
```

Comment:
Maybe we can find a better word for data here. Values?

Reply:
Items? "Associated data" was copied from the Generic Sensor API spec; WHATWG specs are more repetitive and say "A [=foo=] has an associated `<dfn ...>`bar`</dfn>`. A [=foo=] has an associated `<dfn ...>`baz`</dfn>`" that I wanted to avoid.